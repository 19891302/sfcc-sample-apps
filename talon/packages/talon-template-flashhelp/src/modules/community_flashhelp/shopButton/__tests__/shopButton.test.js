import { createElement } from 'lwc';
import ShopButton from 'community_flashhelp/shopButton';

describe('community_flashhelp/shopButton', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-shop-button', { is: ShopButton });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
