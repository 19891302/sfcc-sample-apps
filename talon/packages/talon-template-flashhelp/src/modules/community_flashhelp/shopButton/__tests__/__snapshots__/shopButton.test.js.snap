// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/shopButton renders correctly 1`] = `
<community_flashhelp-shop-button>
  #shadow-root(open)
    <a
      href="https://www.redcross.org/store"
      style="text-decoration: none;"
      target="_blank"
    >
      <span
        style="font-size: 14px; color: rgb(225, 27, 34); font-weight: bold; line-height: 24px;"
      >
        <i
          class="fas fa-shopping-cart"
        />
        <span
          class="icon-label"
        >
          Shop the Red Cross Store
        </span>
      </span>
    </a>
</community_flashhelp-shop-button>
`;
